src/block/icon-list/save.js

```diff
@@ -19,29 +20,47 @@ import { Fragment } from '@wordpress/element'
 import { RichText } from '@wordpress/block-editor'
 
 const save = props => {
-	const { className } = props
+	const { className, attributes } = props
 	const {
-		icon,
-		text,
 		design = '',
 		displayAsGrid = false,
 	} = props.attributes
 
 	const mainClasses = classnames( [
 		className,
 		'ugb-icon-list--v2',
-		`ugb-icon--icon-${ icon }`,
 	], applyFilters( 'stackable.icon-list.mainclasses', {
 		'ugb-icon-list--display-grid': displayAsGrid,
 	}, design, props ) )
 
-	return (
-		<BlockContainer.Save className={ mainClasses } blockProps={ props } render={ () => (
-			<Fragment>
+	const ListItem = range( 1, 7 ).map( index => {
+		const icon = attributes[ `icon${ index }` ]
+		const text = attributes[ `text${ index }` ]
+		return (
+			<li
+				className={ `ugb-icon-list--item${ index }` }
```

Comment:
Follow [BEM](http://getbem.com/introduction/)

```suggestion
				className={ `ugb-icon-list__item${ index }` }
```